submissions/7SergSmith7/Friends-app/app.js

```diff
@@ -0,0 +1,143 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
+
+const USERS_URL =
+  "https://randomuser.me/api/?inc=gender,phone,picture,dob,location,name&results=25";
+
+const contactsField = document.querySelector(".contacts-field");
+const optionsMenu = document.querySelector(".options");
+const searchInput = document.querySelector(".options__search__inpit");
+let contactsData;
+let tempContacts;
+let filter = "";
```

Comment:
Is it should be global? 🤔

Reply:
It seemed logical to me.
How to do it better?

Reply:
You use it's value only in `onOptionsClick`, basically.

So why it should be global? Or I am missing something?

Reply:
It is used in the option reset function